app/mm-redux/reducers/entities/channels.ts

```diff
@@ -543,6 +543,22 @@ function stats(state: RelationOneToOne<Channel, ChannelStats> = {}, action: Gene
 
 function groupsAssociatedToChannel(state: any = {}, action: GenericAction) {
     switch (action.type) {
+    case GroupTypes.RECEIVED_ALL_GROUPS_ASSOCIATED_TO_CHANNELS_IN_TEAM: {
+        const {groupsByChannelId} = action.data;
+        const nextState = {...state};
+
+        for (const channelID of Object.keys(groupsByChannelId)) {
+            if (groupsByChannelId[channelID]) {
```

Comment:
@catalintomai 
Do you happen to know if I can remove Line 551 as suggested by Miguel?

Reply:
@hahmadia, I agree that it is very unlikely that the value will ever be null/undefined. OTOH, I think it is generally be better to be paranoid and check as much as we can, given that code evolves and it could be that in the future the server code could change. So I'm more of the opinion that we should keep the check even if it seems superfluous today.